tasks.go

```diff
@@ -54,7 +54,10 @@ func RunAllTasks(runner Runner, path string, workers int) {
 // FIXME: GetAllTasks should not need to know about basepath.
 func GetAllTasks(runner Runner, basepath string) <-chan Task {
 	var (
-		resources = []string{"deploymentconfigs", "pods", "services", "events"}
+		resources = []string{
+			"deploymentconfigs", "pods", "services", "events",
+			"persistentvolumes", "persistentvolumeclaims",
```

Comment:
yeah, I overlooked that alright! Similar to the diagnostics task in that case then.